layouts/partials/styles.html

```diff
@@ -15,5 +15,9 @@
 {{ if $.Param "mobilemenu" -}}
 <link rel="stylesheet" href="{{ $mobile_style.RelPermalink }}" media="screen">
 {{ end -}}
+{{ if $.Param "math" -}}
```

Comment:
I intend to conditionally load the CSS file here. However, this trick doesn't work if I set the `math` property to `true` in page params but leave that in site params to remain `false`, and the math typesetting could not be rendered properly.

I've read the Hugo docs. about variables and params, but I still have no clue so far. Any hints or suggestions will be appreciated.

Reply:
> I intend to conditionally load the CSS file here. However, this trick doesn't work if I set the `math` property to `true` in page params but leave that in site params to remain `false`, and the math typesetting could not be rendered properly.
> 
> I've read the Hugo docs. about variables and params, but I still have no clue so far. Any hints or suggestions will be appreciated.

Other proposed changes have been addressed except this.

Reply:
All right, I see; thanks for pointing it out. I will fix other stuff too ; )